test.md

```diff
@@ -16,13 +16,24 @@ Assertions
 These assertions will check the supplied property, and then clear that state from the configuration.
 In this way, tests can be written as a serious of setup, execute, assert cycles which leaves the configuration empty on success.
 
+We'll make `Assertion` a subsort of `Instr`, so that we can consume them with `trap` statements.
+
```

Comment:
If you want you can add a short sentence on why we want to consume traps the way we do here, took me a bit of thinking to see that it's needed for the select tests.

Reply:
I added some more documentation of this.